src/features/form/components/FormFilloutPage.module.css

```
@@ -56,6 +56,75 @@
+.navigationGroup {
+	display: flex;
+	align-items: center;
+	gap: 0.4rem;
+}
+
+/* Uniform width for the nav buttons. */
+.navigationGroup button {
+	min-width: 5rem;
+}
+
+/* Segmented undo/redo pill*/
```

Stylelint (17.14.1)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)